apps/storybook/src/SvgElement.stories.module.css

```diff
@@ -0,0 +1,12 @@
+.rect {
+  fill: teal;
+  fill-opacity: 0.2;
+  stroke: teal;
+  stroke-width: 2;
+  pointer-events: auto;
```

Comment:
Note this blocks panning/zooming when hovering the rectangle

Reply:
Yep, we'll have to find a solution for this at some point...